DOCUMENTATION.md

```diff
@@ -368,6 +368,68 @@ If the request is not successful, the client returns an `HTTPError` containing t
 |`500`|`[{`<br>`"error": "Exception",`<br>`"message": "Internal server error"`<br>`}]`|Notify was unable to process the request, resend your notification.|
 
 
+### Send a precompiled Letter
```

Comment:
All headings should be moved up one level e.g. line 371 should be an H2, line 374 should be an H3, and so on